site/en/blog/devtools-tips-container-queries/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: >
+  DevTools Tips: How to inspect CSS container queries
+description: >
+  Inspect and debug CSS container queries with DevTools.
+layout: 'layouts/blog-post.njk'
+date: 2022-06-30
+authors:
+  - sofiayem
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7fid3Zx57cRSpIndGA2M.png'
+alt: >
+  DevTools Tips hero logo
+tags:
+  - css
+  - devtools
+  - devtools-tips
+---
+
+[CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) are the new CSS feature that web developers are excited about. It may trigger a tectonic shift in the design approach of website layouts, from page-based responsive design to container-based [responsive design](https://web.dev/new-responsive/#responsive-to-the-container).
```

Comment:
```suggestion
[CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) is the new CSS feature that web developers are excited about. It may trigger a revolution in web design—from page-based responsive design to container-based [responsive design](https://web.dev/new-responsive/#responsive-to-the-container).
```

Reply:
we had container queries "are", then "It" referring to CQs. It's really "The Container Queries specification" or "The container queries feature", so I think singular is likely correct (even though I also keep using plural as it looks more correct).

Reply:
> ```suggestion
> [CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) is the new CSS feature that web developers are excited about. It may trigger a revolution in web design—from page-based responsive design to container-based [responsive design](https://web.dev/new-responsive/#responsive-to-the-container).
> ```

Reply:
> ```suggestion
> [CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) is the new CSS feature that web developers are excited about. It may trigger a revolution in web design—from page-based responsive design to container-based [responsive design](https://web.dev/new-responsive/#responsive-to-the-container).
> ```